resource/user/user.go

```diff
@@ -93,10 +104,17 @@ func (u *User) Check(resource.Renderer) (resource.TaskStatus, error) {
 
 			switch {
 			case userByName != nil:
-				status.Level = resource.StatusFatal
```

Comment:
It should not have been fatal. With state=present, if the user exists, we do not change.